kubernetes-manifests/redis.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       labels:
         app: redis-cart
     spec:
+      serviceAccountName: default
```

Comment:
**Recap from 1-on-1 with Mathieu:** We initially needed this when we were using `sed` (instead of `kustomize`). We don't need this anymore (since we think `kustomize` won't care if it's initially empty), but we might as well leave this to be consistent with other base YAML files.